packages/dumb/src/common/tooltips/stat-tooltip.tsx

```diff
@@ -27,74 +27,34 @@
 
 import React, { FunctionComponent } from 'react'
 import ReactMarkdown from 'react-markdown'
-
-import colors from '../../styles/colors.css'
 import styles from './stat-tooltip.css'
-import Tooltip from './tooltip'
+import { Tooltip, withStyles } from '@material-ui/core'
```

Comment:
Default import instead for MUI components : `import Tooltip from '@material-ui/core/Tooltip'`